### PR TITLE
#273 Support for reading XML schema file in classpath

### PR DESCRIPTION
**Pull request by carlosmiranda**
#273:

* Updated jcabi-xml to `0.17.1`, which supports lookup of schema file in classpath and fixes some critical bugs.
* Added test for scenario where XML schema is in classpath.

### CODE REVIEW

**Comment by davvd** [comment]
@carlosmiranda Many thanks for the PR, let me find a reviewer for it

**Comment by davvd** [comment]
@pinaf please review, thanks

File: qulice-xml/src/test/java/com/qulice/xml/XmlValidatorTest.java [hunk 1]
```diff
@@ -260,4 +260,31 @@ public void passesValidationIfCommentsAreBeforeParentTag()
...
+                .append("\" ")
+                .append("xsi:schemaLocation=\"")
+                .append("http://qulice.com/test/schema ")
+                .append("test-classpath-schema.xsd")
+                .append("\">")
+                .append("<bar>333</bar>")
+                .append("<baz>444</baz>")
+                .append("</foo>").toString()
+        );
+        final Validator validator = new XmlValidator();
```
Comment (pinaf) [inline qulice-xml/src/test/java/com/qulice/xml/XmlValidatorTest.java]: @carlosmiranda we can inline this

**Comment by pinaf** [comment]
@carlosmiranda 1 comment

**Comment by carlosmiranda** [comment]
@pinaf updated, please check.

**Comment by pinaf** [comment]
@rultor merge

File: qulice-xml/pom.xml [hunk 2]
```diff
@@ -61,6 +61,7 @@
         <dependency>
             <groupId>com.jcabi</groupId>
             <artifactId>jcabi-xml</artifactId>
+            <version>0.17.1</version>
```
Comment (krzyk) [inline qulice-xml/pom.xml]: @carlosmiranda jcabi-xml is also defined in qulice-maven-plugin and the one there will be used in the end. So a dependencyManagement section in the parent pom is needed.

**Comment by krzyk** [comment]
@carlosmiranda nice, one comment from me

File: qulice-xml/pom.xml (hunk 2 above)
Comment (carlosmiranda) [inline qulice-xml/pom.xml]: @krzyk Right, thanks for the heads up!

Comment (carlosmiranda) [inline qulice-xml/pom.xml]: @krzyk actually parent pom already uses this version. So we can remove this.

**Comment by carlosmiranda** [comment]
@krzyk Fixed, please check again.

**Comment by krzyk** [comment]
@carlosmiranda thanks

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by davvd** [comment]
@pinaf Thanks a lot, I just topped your account for **18 mins**, transaction ID `AP-98J87835TA654074H`, total time was 24 hours and 27 mins. review comments (c=3) added as a bonus. +18 added to your rating, current score is: [+5759](http://www.netbout.com/b/35128?open=rating)

**Comment by davvd** [comment]
@rultor deploy now pls

**Pull request merged**